recipe/run_conda_forge_build_setup_linux

```diff
@@ -50,6 +50,10 @@ if [[ ! -z "$CUDA_HOME" && -d /usr/local/cuda-9.2 ]]; then
   sudo yum install -y cuda-compat-10-0.$(uname -m) ;
   # note: this path is added to ldconfig in linux-anvil-cuda:9.2
   if [[ ! -f "/usr/local/cuda-10.0/compat/libcuda.so" ]]; then exit 1; fi
+
+  # Export CONDA_OVERRIDE_CUDA to allow __cuda to be detected on CI systems without GPUs
+  CUDA_VERSION="$(cat ${CI_SUPPORT}/${CONFIG}.yml | shyaml get-value cuda_compiler_version.0 None)"
```

Comment:
The default value should be `""` (empty string), according to the [conda docs](https://docs.conda.io/projects/conda/en/latest/user-guide/tasks/manage-virtual.html#overriding-detected-packages). I think this might fix that.
```suggestion
  CUDA_VERSION="$(cat ${CI_SUPPORT}/${CONFIG}.yml | shyaml get-value cuda_compiler_version.0 '')"
```